client/js/components/make-sticky.js

```diff
@@ -0,0 +1,24 @@
+import {nodeList, featureTest} from '../util';
+import debounce from 'lodash.debounce';
+
+const makeSticky = (els) => {
+  if (featureTest('position', 'sticky')) {
+    const elements = nodeList(els);
+    const applyPositioning = () => {
+      if (document.readyState === 'complete') {
+        elements.map((e) => {
+          if (e.offsetHeight > window.innerHeight) {
+            e.classList.remove('sticky-applied');
+          } else {
+            e.classList.add('sticky-applied');
+          }
+        });
+      }
+    };
+
+    document.addEventListener('readystatechange', applyPositioning);
+    window.addEventListener('resize', debounce(applyPositioning, 500));
```

Comment:
Could we be using the [`requestAnimationFrame`](https://www.html5rocks.com/en/tutorials/speed/animations/) for better perf here?